src/routes/(home)/Discover.svelte

```diff
@@ -0,0 +1,202 @@
+<script lang="ts">
+	import type { ComponentType } from 'svelte';
+	import { inview } from 'svelte-inview';
+
+	import FootBallIcon from '~icons/bx/ball';
+	import BasketballIcon from '~icons/bx/basketball';
+	import CodeIcon from '~icons/bx/code';
+	import GameIcon from '~icons/bx/game';
+	import MathIcon from '~icons/bx/math';
+	import ArtIcon from '~icons/bx/paint';
+	import RocketIcon from '~icons/bx/rocket';
+
+	const fakeClubs = [
+		'Not a club',
+		'Still not a club',
+		'This is noting',
+		"Don't mind me",
+		'Wow, so empty',
+		'Is this a club',
+		'No club here'
+	];
+
+	const icons: ComponentType[] = [
+		FootBallIcon,
+		BasketballIcon,
+		MathIcon,
+		GameIcon,
+		CodeIcon,
+		ArtIcon,
+		RocketIcon
+	];
+
+	const realClubs: {
+		name: string;
+		icon: ComponentType;
+		color: string;
+	}[] = [
+		{
+			name: 'Football club',
+			icon: FootBallIcon,
+			color: '#a85a32'
+		},
+		{
+			name: 'Sports club',
+			icon: BasketballIcon,
+			color: '#32a883'
+		},
+		{
+			name: 'Math club',
+			icon: MathIcon,
+			color: '#307fbf'
+		},
+		{
+			name: 'Board game club',
+			icon: GameIcon,
+			color: '#ab9b0e'
+		},
+		{
+			name: 'Coding club',
+			icon: CodeIcon,
+			color: 'var(--accent)'
+		},
+		{
+			name: 'Art club',
+			icon: ArtIcon,
+			color: '#6e30a1'
+		},
+		{
+			name: 'Rocket club',
+			icon: RocketIcon,
+			color: '#ba2575'
+		}
+	];
+
+	let clubDisplayThing: {
+		name: string;
+		color: string;
+		icon: ComponentType;
+		real: boolean;
+	}[] = [];
+
+	//Assemble the items
+	for (let i = 0; i < 100; i++) {
+		//Should we do a real club display thing?
+
+		clubDisplayThing.push({
+			name: fakeClubs[Math.floor(Math.random() * fakeClubs.length)],
+			icon: icons[Math.floor(Math.random() * icons.length)],
+			color: { ...realClubs[Math.floor(Math.random() * realClubs.length)] }.color,
+			real: false
+		});
+	}
+
+	for (let i = 0; i < realClubs.length; i++) {
```

Comment:
@LeoDog896 can you take a look at an issue I'm having? 
On the page in the Find whats important section, the icons are not associated with the club, but I feel like this code should keep the icons in check. Lmk what you find.